test/libsolidity/semanticTests/array/copying/copy_calldata_array_to_mapping.sol

```diff
@@ -0,0 +1,18 @@
+pragma abicoder v2;
+
+contract C {
+    mapping (uint => uint8[][]) m;
+
+    uint8[][] s;
+
+    function from_calldata(uint8[][] calldata _a) public returns (uint8[][] memory) {
```

Comment:
Why not have this function in file `copy_array_to_mapping.sol` ?

Reply:
Also, I think we could omit the `copy` prefix to the file names since they are inside `array/copying`.

Reply:
We cannot have this function in `copy_array_to_mapping.sol` since copying nested calldata dynamic arrays to storage is not implemented in the old code generator. However, I can adjust filenames as you suggest.

Reply:
I already adjusted the filenames. By the way, I also rebased PR on top of the develop.